src/ert/config/parsing/config_schema_deprecations.py

```diff
@@ -224,4 +224,16 @@
         "the future. Replace by NUM_CPU on a single compute node.",
         check=lambda line: "NUM_NODES" in line,
     ),
+    DeprecationInfo(
+        keyword="QUEUE_OPTION",
+        message="MEMORY_PER_JOB as QUEUE_OPTION to TORQUE is deprecated and will be removed in "
+        "the future. Replace by REALIZATION_MEMORY.",
+        check=lambda line: "MEMORY_PER_JOB" in line,
+    ),
+    DeprecationInfo(
+        keyword="QUEUE_OPTION",
+        message="MEMORY as QUEUE_OPTION to SLURM is deprecated and will be removed in "
+        "the future. Replace by REALIZATION_MEMORY.",
+        check=lambda line: "MEMORY" in line,
```

Comment:
Will this also issue a deprecation warning for the `REALIZATION_MEMORY` keyword?

Reply:
No, you would have to write `QUEUE_OPTION * REALIZATION_MEMORY 50mb` for this to kick in, but that is in any case invalid.

Reply:
Right, it is not a queue_option but a general keyword.